frontend/routes/components/screenshotDialog/components/tools/tools.component.tsx

```diff
@@ -103,26 +182,57 @@ export class Tools extends React.PureComponent<IProps, any> {
 		<StyledButton onClick={this.props.onSave} color="secondary" variant="raised">Save</StyledButton>
 	));
 
-	public handleToolClick = (type, callback?) => () => {
-		this.setState({ activeTool: type }, callback);
+	public handleUndo = () => () => {
+		this.props.onUndo();
+	}
+
+	public handleRedo = () => () => {
```

Comment:
I believe we can write it like that: 
```
public handleRedo = () => {
   this.props.onRedo();
}
```
and then just pass it `action={this.handleRedo}` in `TooltipButton `.

The same goes for `handleUndo `